src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -156,16 +166,41 @@ export class SortFilterBar
     `;
   }
 
+  willUpdate(changed: PropertyValues) {
+    console.log(
```

Comment:
Question: Are we planning to leave this `console.log` in?
Suggestion: `// eslint-disable-next-line no-console`

Reply:
Oops, that can be removed. Thanks!